cobalt/watchdog/watchdog_test.cc

```diff
@@ -28,6 +28,7 @@ namespace {
 
 const char kWatchdogViolationsJson[] = "watchdog_test.json";
 const int64_t kWatchdogMonitorFrequency = 100000;
```

Comment:
Can you rewrite this as a multiple of `kSbTimeMillisecond` instead? That way it's clear what unit the variable is in.

Reply:
Everything is in terms of `int64_t` and microseconds and in watchdog.cc/h there are comments that specify that. I think there's value in switching from `int64_t` to `SbTime`, but I would want that change to be all over for watchdog for it to be consistent.